web/modules/weather_data/src/Service/AlertUtility.php

```diff
@@ -954,7 +931,7 @@ public static function getPlacesFromAlertDescription($alert)
         //
         // So if we have a line that matches, we should try parsing it.
         preg_match(
-            "/IN \S+ THIS \S+ INCLUDES \d+ COUNTIES$/sim",
+            "/IN \S+ (THIS|THE NEW) \S+ INCLUDES \d+ COUNTIES$/sim",
```

Comment:
Oh, oops. Good catch! One of the alerts had place information in it but the heading said `IN THE NEW WATCH` instead of our expected `IN THIS WATCH`. It looks like that alert has been replaced now, but this was to fix that and I totally forgot to mention it in the PR body.

Reply:
Here's the watch: https://mesonet.agron.iastate.edu/wx/afos/p.php?pil=WCNJAX&e=202408051001